addons/l10n_cl/data/account_tax_data.xml

```diff
@@ -175,12 +175,12 @@
             (0,0, {
                 'factor_percent': 100,
                 'repartition_type': 'base',
-                'plus_report_line_ids': [ref('tax_report_compras_activo_fijo')],
```

Comment:
@Danisan no way in stable: this will give problems for all existing reports.  Change the formulas instead.

Reply:
solved in meeting. We keep this since we realized it does not impact.